service/cloudwatch/serializers.go

```diff
@@ -2450,6 +2450,7 @@ func (m *awsAwsquery_serializeOpUntagResource) HandleSerialize(ctx context.Conte
 }
 func awsAwsquery_serializeDocumentAlarmNames(v []string, value query.Value) error {
 	if len(v) == 0 {
+		value.Array("member").Empty()
 		return nil
 	}
 	array := value.Array("member")
```

Comment:
Or  maybe this makes more sense?

```suggestion
	array := value.Array("member")
	if len(v) == 0 {
		array.Empty()
		return nil
	}
```